test/example/visitor-basic.ll

```diff
@@ -20,7 +22,7 @@ entry:
   %v2 = call i32 @llvm.umax.i32(i32 %v1, i32 %q)
   %t = call i8 (...) @xd.itrunc.i8(i32 %v2)
   call void (...) @xd.write(i8 %t)
-  call void (...) @xd.write.vararg(i8 %t, i32 %v2)
+  call void (...) @xd.write.vararg(i8 %t, i32 %v2, i32 %q)
```

Comment:
This example would have crashed before this fix?

Reply:
Yes, assuming the additional change in the visitor code.